libraries/AP_HAL_ChibiOS/system.cpp

```diff
@@ -159,6 +159,11 @@ uint32_t millis()
     return hrt_millis32();
 }
 
+uint16_t millis16()
```

Comment:
Because millis64 was being called in the base implementation, and that does a division.

Moving to masking millis() in the base class means we don't need to override it here now.